### PR TITLE
Fix to allow animals loaded in RDBs to make sounds

### PR DESCRIPTION
**Pull request by jeniferirwin**

### CODE REVIEW

File: Assets/Scripts/Utility/RMBLayout.cs [hunk 1]
```diff
@@ -1001,7 +1001,7 @@ private static bool IsBulletinBoard(uint modelID)
             return modelID == BulletinBoardModelID;
         }
 
-        private static void AddAnimalAudioSource(GameObject go, int record)
+        public static void AddAnimalAudioSource(GameObject go, int record)
```
Comment (df-ferital) [inline Assets/Scripts/Utility/RMBLayout.cs:1004]: This method should be moved outside RMBLayout since it is now also used for RDB layout.

Comment (KABoissonneault) [reply Assets/Scripts/Utility/RMBLayout.cs:1004]: @df-ferital Do you have any suggestions? I tried looking a bit, and couldn't find a good place. Maybe `GameObjectHelper`?

Comment (df-ferital) [reply Assets/Scripts/Utility/RMBLayout.cs:1004]: Yes, I suppose `GameObjectHelper` would be the best place.

Comment (jeniferirwin) [reply Assets/Scripts/Utility/RMBLayout.cs:1004]: I am new to the DFU scene. This change you're discussing - is it a change that I should be making, or is it something that more experienced DFU programmers will make after the pull request is merged? If I'm the one that should be making this change, should I cancel this pull request and re-do it? Thanks in advance for your time!

Comment (KABoissonneault) [reply Assets/Scripts/Utility/RMBLayout.cs:1004]: You should do it, but you don't have to cancel. You can make a new commit, push the commit to the same branch (jeniferirwin:master), and the PR will update accordingly

Comment (jeniferirwin) [reply Assets/Scripts/Utility/RMBLayout.cs]: I've made the change and added the commit. Thanks for the explanation!

**Review by KABoissonneault — APPROVED** [review]

**Review by df-ferital — APPROVED** [review]

**Comment by Interkarma** [comment]
Thank you for the refinement @jeniferirwin. And thank you @KABoissonneault and @df-ferital for help with review. :)

**Pull request merged**